corehq/apps/users/migrations/0001_add_location_permission.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-03-09 02:05
+from __future__ import unicode_literals
+from corehq.sql_db.operations import HqRunPython
+from corehq.apps.users.management.commands.add_location_permission import update_all_roles
+
+from django.db import migrations
+
+
+def migrate(apps, schema_editor):
+    update_all_roles()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        HqRunPython(migrate),
```

Comment:
Yeah. it seems reasonable to not want to run this on a bunch of different places (especially NIC)

Was just thinking that this migration isn't something that will need to be re-run on every new environment/test and this was one of the things that slowed down accounting so hard. I suppose we could change this to a no-op migration after it's been run everywhere